Dockerfile

```diff
@@ -1,30 +1,33 @@
 FROM golang:1.14-alpine AS builder
```

Comment:
Please don't mix ubuntu for run and alpine for build it's known to cause errors.

Reply:
It will go away in the next PR. This one was only about changing runtime. Btw, `CGO_ENABLED=0` kinda solves a problem with building/running in glibc/musl-libc envs.